### PR TITLE
Added Resources from aaron-s and Slack

### PR DESCRIPTION
**Pull request by CryptoJones**
Additional nodes to the resources.yml file.

### CODE REVIEW

File: resources.yml [hunk 1]
```diff
@@ -11660,8 +11660,204 @@
...
+  notes: Random Data Generator and API Mocking Tool (Json, CSV, SQL, Excel)
+
+  - name: Array Ipsum
+  url: https://array-ipsum.com
+  category: Tools
+  languages: ["JavaScript"]
+  paid: false
+  notes: Sample Arrays
+
+  - name: Array Explorer by Sarah Drasner
```
[the remark this answers is not in the document]
Comment (CryptoJones) [reply resources.yml]: Fixed.

File: resources.yml [hunk 2]
```diff
@@ -11660,8 +11660,204 @@
...
+  url: https://sdras.github.io/array-explorer/
+  category: Tools
+  languages: ["JavaScript"]
+  paid: false
+  notes: 
+
+- name: Twenty Questions to Ask Before Joining a Startup
+  url: https://hharnisc.github.io/2018/11/25/twenty-questions-to-ask-before-joining-a-startup.html
+  category: Blog Post
```
[the remark this answers is not in the document]
Comment (CryptoJones) [reply resources.yml]: Done

File: resources.yml [hunk 3]
```diff
@@ -11660,8 +11660,204 @@
...
+- name: The Tangled Web - A Guide to Securing Modern Web Applications
+  url: https://repo.zenk-security.com/Techniques%20d.attaques%20%20.%20%20Failles/The%20Tagled%20Web%20A%20Guide%20to%20Securing%20Modern%20Web%20Applications.pdf
+  category: Books
+  languages: []
+  paid: false
+  notes: By Michal Zalewski
+
+- name: Android Application Security Series
+  url: https://manifestsecurity.com/android-application-security/
+  category: Blog Posts
```
[the remark this answers is not in the document]
Comment (CryptoJones) [reply resources.yml]: Fixed

File: resources.yml [hunk 4]
```diff
@@ -11660,8 +11660,204 @@
...
+  notes: 
+
+- name: The HackerRank Interview Preparation Kit
+  url: https://www.hackerrank.com/interview/interview-preparation-kit 
+  category: Training
+  languages: []
+  paid: false
+  notes: Include all major Languages
+
+- name: Algo VPN is a set of Ansible scripts that simplify the setup of a personal IPSEC VPN.
```
[the remark this answers is not in the document]
Comment (CryptoJones) [reply resources.yml]: Holy crap, how did I miss that?

Fixed.

File: resources.yml [hunk 5]
```diff
@@ -11660,8 +11660,204 @@
...
+  notes: Streisand sets up a new server running your choice of WireGuard, OpenConnect, OpenSSH, OpenVPN, Shadowsocks, sslh, Stunnel, or a Tor bridge.
+
+- name: Why Junior Developers Should Be Giving Presentations
+  url: https://medium.com/@melindagolden/why-junior-developers-should-be-giving-presentations-39bf7f49a858
+  category: Blog Posts
+  languages: []
+  paid: false
+  notes: 
+
+- name: Training Application
```
[the remark this answers is not in the document]
Comment (CryptoJones) [reply resources.yml]: I should have check the <title> tag. It's "Mimo: Learn to code."

Fixed.

File: resources.yml [hunk 6]
```diff
@@ -11660,8 +11660,204 @@
...
+- name: Accessibility - Rethinking First Look Media’s Design and Development Workflow
+  url: https://tech.firstlook.media/accessibility-rethinking-first-look-media-s-design-and-development-workflow
+  category: Blog Post
+  languages: []
+  paid: false
+  notes: 
+
+- name: Online IDE for web applications.
+  url: https://www.stackblitz.com
+  category:
```
[the remark this answers is not in the document]
Comment (CryptoJones) [reply resources.yml]: Fixed.

File: resources.yml [hunk 7]
```diff
@@ -11660,8 +11660,204 @@
...
+
+- name: Twenty Questions to Ask Before Joining a Startup
+  url: https://hharnisc.github.io/2018/11/25/twenty-questions-to-ask-before-joining-a-startup.html
+  category: Blog Post
+  languages: []
+  paid: false
+  notes: 
+
+- name: Next Generation Training for IT
+  url:
```
[the remark this answers is not in the document]
Comment (CryptoJones) [reply resources.yml]: And THIS is why we have code reviews.

Fixed.

File: resources.yml [hunk 8]
```diff
@@ -11660,8 +11660,204 @@
...
 - name: Learning C#
-  url: https://www.cryptospace.com/~akclark/Learning%20CSharp
+  url: https://www.cryptospace.com/~akclark/Learning%20CSharp/
   category: Microsoft
   languages: ["C#"]
   paid: false
-  notes:
+  notes: Original Course by Bob Tabor, from http://www.learnvisualstudio.net
+
+  - name: Mockaroo
```
[the remark this answers is not in the document]
Comment (CryptoJones) [reply resources.yml]: Fixed for three instances of " - name:"

File: resources.yml [hunk 9]
```diff
@@ -11660,8 +11646,197 @@
...
+- name: The Art of Software Security Assessment
+  url: https://leaksource.files.wordpress.com/2014/08/the-art-of-software-security-assessment.pdf
+  category: Books
+  languages: []
+  paid: false
+  notes: By Mark Dowd, John McDonald, and Justin Schuh
+
+- name: LinkedIn Profile Tips for Bootcamp Grads
+  url: https://www.coursereport.com/blog/linkedin-profile-tips-for-coding-bootcamp-grads 
+  category: Social Media
```
Comment (aaron-suarez) [inline resources.yml]: I think we're going to have to address this when we do the data consolidation thing, but I'm cool merging it as is for now.

**Pull request merged**